src/interface/productInterface.ts

```diff
@@ -0,0 +1,42 @@
+export interface IProduct {
+  id: string,
+  name: string,
+  code: string,
+  category: string,
+  type: string,
+  popularity: number,
+  quality: {
```

Comment:
we tolk about `quantity` interface, but i don't see it

Reply:
fixed